docs/src/main/asciidoc/compose-dev-services.adoc

```diff
@@ -0,0 +1,632 @@
+
+[source, yaml]
+----
+services:
+  db:
+    image: postgres:17
+    profiles:
+      - dev
+      - local
+    ports:
+      - '5432'
+    environment:
+      POSTGRES_USER: myuser
+      POSTGRES_PASSWORD: mypassword
+      POSTGRES_DB: mydb
+----
+
+[source, properties]
+----
+%test.quarkus.compose.devservices.profiles=dev,local
+----
+
+=== Ignoring Services
+
+You can configure Compose Dev Services to ignore specific services by adding the `io.quarkus.devservices.compose.ignore` label to the service in your `compose.yml` file:
+
+[source, yaml]
+----
+services:
+  db:
+    image: postgres:17
+    labels:
+      io.quarkus.devservices.compose.ignore: true
+    ports:
+      - '5432'
+    environment:
+      POSTGRES_USER: myuser
+      POSTGRES_PASSWORD: mypassword
+      POSTGRES_DB: mydb
+----
+
+When a service has this label set to `true`, it will be excluded from service discovery, and extensions will not use it for Dev Services.
+
+== Service Readiness
+
+Compose Dev Services provides several ways to ensure services are ready before your application tries to use them.
+
+=== Health Checks
+
+Compose Dev Services respects the health checks defined in your `compose.yml` file.
+It's recommended to configure health checks for your services to ensure they are ready before your application tries to use them:
+
+[source, yaml]
+----
+services:
+  db:
+    image: postgres:17
+    healthcheck:
+      test: pg_isready -U myuser -d mydb
+      interval: 5s
+      timeout: 3s
+      retries: 3
+    ports:
+      - '5432'
+    environment:
+      POSTGRES_USER: myuser
+      POSTGRES_PASSWORD: mypassword
+      POSTGRES_DB: mydb
+----
+
+=== Wait for Logs
+
+You can configure Compose Dev Services to wait for specific log messages to appear in the container logs before considering a service ready.
+This is useful for services that don't provide health checks but log a message when they're ready.
+
+To wait for a specific log message, add a label with the prefix `io.quarkus.devservices.compose.wait_for.logs` to your service:
+
+[source, yaml]
+----
+services:
+  app:
+    image: my-application:latest
+    ports:
+      - '8080'
+    labels:
+      io.quarkus.devservices.compose.wait_for.logs: .*Server is now running.*
+----
+
+You can also specify how many times a log message should appear by setting the numeric suffix value:
+`io.quarkus.devservices.compose.wait_for.logs.3: .*Worker started.*`
+
+=== Wait for Ports
+
+By default, Compose Dev Services waits for all exposed ports to be available before considering a service ready.
+This behavior can be customized using labels:
+
+[source, yaml]
+----
+services:
+  app:
+    image: my-application:latest
+    ports:
+      - '8080'
+    labels:
+      # Disable waiting for ports
+      io.quarkus.devservices.compose.wait_for.ports.disable: true
+      # Or set a custom timeout for port waiting
+      io.quarkus.devservices.compose.wait_for.ports.timeout: "30s"
+----
```

Comment:
This is a small comment, but since it's possible to copy this code directly from the docs page, wouldn't it be better to have an example that isn't using conflicting config settings? E.g. If I disable wait for ports, the timeout doesn't make any sense. My suggestion is to change the `io.quarkus.devservices.compose.wait_for.ports.disable: false` and in the comment above, change to `true` if you want to disable wait for ports.